src/App.js

```diff
@@ -232,7 +229,13 @@ const returnLoadedWalletComponent = (Component, props, rest) => {
   } else {
     // If was closed and is loaded we need to redirect to locked screen
     if (hathorLib.wallet.wasClosed()) {
-      return <Redirect to={{ pathname: '/locked/' }} />;
+      if (isServerScreen) {
+        // If there is a server problem before the wallet is unlocked
+        // the server screen must be the priority over the locked screen
+        return returnDefaultComponent(Component, props);
```

Comment:
Not sure when this would happen. If wallet was closed and the user was on the server screen, t won't be on the server screen next time it reopens.

Also, he should only be able to go to the server screen after unlocking the wallet.